src/addon/cc-addon-option-form.js

```diff
@@ -0,0 +1,124 @@
+import '../addon/cc-addon-option.js';
+import '../atoms/cc-button.js';
+import '../molecules/cc-block.js';
+import { css, html, LitElement } from 'lit-element';
+import { dispatchCustomEvent } from '../lib/events.js';
+import { i18n } from '../lib/i18n.js';
+
+/**
+ * A component that displays a form of `<cc-addon-option>`.
+ *
+ * * 🎨 default CSS display: `block`
+ * <br>
+ * 🧐 [component's source code on GitHub](https://github.com/CleverCloud/clever-components/blob/master/src/addon/cc-addon-option-form.js)
+ *
+ * ## Details
+ *
+ * * Emits a `submit` event when the form is submited. The emitted event is an object of options state (enabled / disabled).
+ *
+ * ## Type definitions
+ *
+ * ```js
+ * interface Option {
+ *   title: String,
+ *   logo: String,
+ *   description: DOM,
+ *   name: String,
+ *   enabled: Boolean
+ * }
+ *
+ * interface Options {
+ *   optionName1: Boolean,
+ *   optionName2: Boolean,
```

Comment:
I think there's another way to describe this in TypeScript interfaces. Here it seems like a strict `optionName1`.